mmocr/apis/utils.py

```diff
@@ -45,17 +45,19 @@ def replace_image_to_tensor(cfg, set_types=None):
 
 
 def update_pipeline_recog(cfg, idx=None):
-    warning_msg = 'Remove "MultiRotateAugOCR" to support batch ' + \
-        'inference since samples_per_gpu > 1.'
+    warning_msg = 'Set "rotate_degrees=[0]" (just one degree) to support ' + \
+        'batch inference since samples_per_gpu > 1.'
     if idx is None:
         if cfg.get('pipeline',
                    None) and cfg.pipeline[1].type == 'MultiRotateAugOCR':
-            warnings.warn(warning_msg)
-            cfg.pipeline = [cfg.pipeline[0], *cfg.pipeline[1].transforms]
+            if len(cfg.pipeline[1].get('rotate_degrees', [0])) != 1:
+                warnings.warn(warning_msg)
+                cfg.pipeline[1]['rotate_degrees'] = [0]
     else:
         if cfg[idx][1].type == 'MultiRotateAugOCR':
-            warnings.warn(warning_msg)
-            cfg[idx] = [cfg[idx][0], *cfg[idx][1].transforms]
+            if len(cfg[idx][1].get('rotate_degrees', [0])) != 1:
+                warnings.warn(warning_msg)
+                cfg[idx][1]['rotate_degrees'] = [0]
 
 
 def disable_text_recog_aug_test(cfg, set_types=None):
```

Comment:
It should be renamed since the propose of `update_pipeline_recog` has been changed

Reply:
All config-related utils in this file should be moved to mmocr/utils/config.py. There are many duplicates.